History.md

```diff
@@ -1,3 +1,9 @@
+unreleased
```

Comment:
Please do it the same as all the previous commits have done it in the 5.0 branch.